### ISSUE TITLE
m3u support

### ISSUE THREAD

**Issue by chromer030** [Palethorn/native-adaptive-streaming#79]
Hi , thank you for this great extension , please add support for m3u extensions , currently by clicking on this urls , browser start to download the playlist.

mainly there are lots of this urls in this repo :

https://github.com/iptv-org/iptv

thank you>

**Comment by Palethorn** [comment]
This is something that crossed my mind a few times. Could be interesting to implement.

**Comment by ForceGT** [comment]
Hey any update on this 
I am working on a project that generates live streaming links 
So it would be great if I can generate an m3u file and your extension can play it

PS: - I did not find players that can play DRM protected content, even VLC doesn't support it yet

**Comment by Palethorn** [comment]
I've been really busy with my regular job and private stuff, so I haven't had much time to work on this project. Hope you understand.

**Comment by ForceGT** [comment]
I completely understand my friend being in a similar situation
I am not that good with Javascript but I will definitely try to contribute to this feature if possible

**Comment by SpeedxThunder001** [comment]
hi, any update on this?

**Comment by Palethorn** [comment]
Hi, I've been working on migrating the extension to manifest V3, but it was missing some features that this extension requres to work properly. I've been waiting on chromium dev team to implement the feature but the report on their system became stale after a year. That was holding me back, and I'm not sure if they'll implement the request at this point. More info https://bugs.chromium.org/p/chromium/issues/detail?id=1262147#c5
I will probably revert to V2 at this point and continue issuing updates until I come up with a better idea.
Manifest V2 is legacy at this point, they only allow existing extensions on chrome store to use V2, and it's a matter of time until they pull the plug on those.

**Comment by chromer030** [comment]
I think we have still too much time till MV3 obligation , as they postponed the phasing out :
> As Manifest V3 approaches full feature parity with Manifest V2, we'll be progressively phasing out Manifest V2. This page specifies the timetable for this deprecation and describes the meaning of each milestone.

![txfDeyLhratHCO1P3wvc](https://user-images.githubusercontent.com/13361552/219855099-bd0d5442-f763-4987-9af5-b42af0d17db9.jpg)

https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/
